docs/release-notes.rst

```diff
@@ -0,0 +1,205 @@
+    - Proxy ARP implementation using a dummy virtual router gateway
+    - Production ready
+  * - Pod to pod connectivity using vrouter
+    - Each pod is connected to another pod using L3 virtual routing.
+      Proxy arp implementation for arp resolution.
+    - Production ready
+  * - Support for DPDK P4 pipeline for Pod to Pod connectivity
+    - A P4 DPDK pipeline for pod to pod connectivity and service load-balancing
+      using TAP interfaces and a P4 DPDK pipeline
+    - Production ready
+  * - Flow connection tracking
+    - Connection tracking and NAT implementation using flow 5-tuple for Service Load Balancing.
+    - Production ready
+  * - Support for Service type “ClusterIP” for UDP and TCP traffic
+    - Support for UDP and TCP services. Fully functional DNS and Kubernetes API services.
+    - Production ready
+  * - Support for Device interfaces like subfunctions and native interfaces like ipvlan and Tap
+    - Subfunctions for ES2100; Tap for DPDK
+    - Production ready
+  * - Execution support in Split Mode with Inframanager running on ACC
+    - Inframanager running on ACC but infraagent on host
+    - Production ready
+  * - Automation scripts for cluster deployment
+    - Example scripts for cluster deployment of Load balancing and CNI add
+    - Production ready
+  * - SRIOV interface support for ES2100
+    - SRIOV support for ES2100 in host mode in addition to CDQ
+    - Engineering preview
+
+Resolved Issues
+===========================
+
+- After deleting and creating multiple test pods, multiple times some of the pods are not
+  getting created, with error "failed to get a CDQ interface for pod: no free resources left" on infraagent.
+- No Readme for TLS certificates and security guide
+- Test pods are not coming to running state, as Policy related errors on inframanager
+- Infraagent is not coming to running state, with the error "Error while parsing Json file"
+- Inframanger restarts/crashes with "Panic occured, runtime error: invalid memory address or nil
+  pointer dereference" error on sending invalid grpc messages from Defensics for fuzz testing.
+- "/opt/p4/k8s/inframanager": No such file or directory" error when running setup_infra.sh in split mode
+- Readme: config-manage for Docker should point to right repo supporting Rocky Linux
+- infra-manager pod not coming up due to problem in cleanup of kustomization.yaml after runnning split mode.
+- scripts/setup_infra.sh doesn't contain right changes to make in split mode for infraagent, inframanger
+  and openssl conf files
+
+Known issues and limitations
+===============================
+
+- The setup_infra.sh automation script, works with the default configuration for certificate paths
+  and artifact paths. Any changes in these paths will render the script unusable.
+  User may need to manually configure and execute instructions mentioned in the script.
+- The `setup_infra_sriov.sh` script doesn't support the `-r` option for remote IP for host IP on ACC.
```

Comment:
SRIOV is an experimental feature. The `setup_infra_sriov.sh` script doesn't support the `-r` option for remote IP for host IP on ACC.